### PR TITLE
Add rhel8 version for postgresql 9.6 and 10

### PR DESCRIPTION
**Pull request by pkubatrh**

### CODE REVIEW

**Comment by hhorak** [comment]
[test]

**Comment by pkubatrh** [comment]
[test-openshift]

**Comment by pkubatrh** [comment]
@praiskup PTAL

File: src/Dockerfile [hunk 1]
```diff
@@ -80,13 +85,14 @@ ENV CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/postgresql \
...
 
+{% if spec.prod != "rhel8" and config.os.id != "fedora" %}
 # When bash is started non-interactively, to run a shell script, for example it
 # looks for this variable and source the content of this file. This will enable
 # the SCL for all scripts without need to do 'scl enable'.
 ENV BASH_ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
     ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
     PROMPT_COMMAND=". ${CONTAINER_SCRIPTS_PATH}/scl_enable"
-
+{% endif %}
```
Comment (praiskup) [inline src/Dockerfile]: line removal

**Comment by praiskup** [comment]
Whitespace issues, is the change in cccp.yaml expected?  Otherwise lgtm.

**Comment by pkubatrh** [comment]
>is the change in cccp.yaml expected? Otherwise lgtm.

afaik the only use for this file is in the Centos pipeline so no reason to have it populated by RHEL/Fedora

[test-openshift]

**Comment by praiskup** [comment]
lgtm

**Comment by pkubatrh** [comment]
Looked a bit into the cccp.yaml change - it changed from original centos7 name into rhel7 name, now changing to centos7 which is correct.

The reason why the cccp.yaml changed before is likely https://github.com/sclorg/container-common-scripts/pull/105 since the generator script now uses any working distgen config combination it can find instead of just hardcoded centos7.

**Comment by pkubatrh** [comment]
One more commit to update common scripts for podman 1.0

[test-openshift]

**Comment by pkubatrh** [comment]
[test]

**Comment by pkubatrh** [comment]
[test]

**Comment by hhorak** [comment]
lgtm

**Pull request merged**